src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+        }
+        Map<String, List<Room>> foundRooms = hotelController.getAvailableRooms(city, person, price,
+                reservedFrom, reservedTo);
+        if (foundRooms.values().isEmpty()) {
+            outputDataInConsole("Room with such parameters not found");
+        } else {
+            outputDataInConsole("The list of found rooms:\n ");
+            outputListInConsole(foundRooms.entrySet());
+            outputSplitLine();
+            bookRoomMenu(reservedFrom, reservedTo, foundRooms);
+            outputDataInConsole("For room reservation please fill out the following form");
+        }
+        return foundRooms;
+    }
+
+    private void bookRoomMenu(Date reservedFrom, Date reservedTo, Map<String, List<Room>> foundRooms) throws IOException {
+        long reserveId = 0L;
+        boolean bookFlaf = true;
+        int numberHotel;
+        Hotel hotelSet = null;
+        Room roomSet = null;
+        while (bookFlaf) {
+            outputDataInConsole("Do you have booking room?");
+            showUserMenu("Booking menu");
+            Integer bookRoom = (Integer) inputDataFromUser("Enter number: ", true,
+                    false, false, false);
+            switch (bookRoom) {
+                case (1):
+                    try {
+                        Integer hotelNumber = (Integer) inputDataFromUser("Enter number hotel: ",
+                                true, false, false, false);
+                        Integer roomNumber = (Integer) inputDataFromUser("Enter number room: ",
+                                true, false, false, false);
+                        int countRooms;
+                        for (Map.Entry<String, List<Room>> e : foundRooms.entrySet()) {
+                            for (Room e1 : e.getValue()) {
+
+                            }
+//                                System.out.println(e.getKey() + " = "+ e1.Out());
+                        }
+
+                        if (hotelNumber > foundRooms.size()) {
+                            outputDataInConsole("Incorrect number, please enter correct number Hotel: ");
+                            continue;
+                        }
+                        if (roomNumber > foundRooms.values().size()) {
+                            outputDataInConsole("Incorrect number, please enter correct number Room: ");
+                            continue;
+                        } else {
+                            String hotel = new ArrayList<>(foundRooms.keySet()).get(0);
+//                            hotelSet = Injector.getHotelController().findHotelByName(hotel).get(--hotelNumber);
+//                            roomSet = foundRooms.get(hotel).get(--roomNumber);
+//                            reserveId = Injector.getReservationController().bookRoom(reservedFrom,
+//                                    reservedTo, roomSet, hotelSet);
+                        }
+                    } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)